src/includes/class-alma-wc-refund.php

```diff
@@ -160,7 +156,6 @@ public function gettext( $translation, $text, $domain ) {
 	 * @return void
 	 */
 	public function admin_notices() {
-
```

Comment:
where `alma_refund_notices` come from ?

Reply:
yep ! and it should be used only in one class (I did it, could you make a review on this commit please https://github.com/alma/alma-woocommerce-gateway/pull/78/commits/01293b74ecb51db74859c8e5b0b2010227814b08 😉 )